graylog2-inputs/src/main/java/org/graylog2/inputs/transports/AmqpTransport.java

```diff
@@ -292,6 +294,15 @@ public ConfigurationRequest getRequestedConfiguration() {
             );
 
             cr.addField(
+                    new BooleanField(
+                            CK_EXCHANGE_BIND,
+                            "Bind to exchange?",
+                            false,
+                            "Binds the queue to the configured exchange. Make sure the exchange exists!"
```

Comment:
Instead of "Make sure the exchange exists!" maybe write "The exchange must already exist."